batch/batch/worker/worker.py

```diff
@@ -2186,7 +2447,7 @@ async def async_main():
     finally:
         try:
             await worker.shutdown()
-            log.info('worker shutdown')
+            log.info('worker shutdown', exc_info=True)
```

Comment:
If exceptions contain user data, then yes it can. We should ensure exceptions do not contain user data, not avoid logging exceptions on shutdown. If a worker unexpectedly shuts down, we need to know why. The lack of this information was an issue for me when debugging this PR.